lib/beaker/module_install_helper.rb

```diff
@@ -144,6 +167,22 @@ def get_module_source_directory(search_in)
     end
     module_source_dir
   end
+
+  def forge_host
+    fh = ENV['BEAKER_FORGE_HOST'] || ENV['forge_host']
```

Comment:
oh wow.

please add a pointer to that (part of beaker's docs, or source if necessary) in the README, so it doesn't get lost again.